index.js

```diff
@@ -69,10 +73,13 @@ async function deleteOlderReleases(keepLatest) {
   try {
     let data = await fetch({
       ...commonOpts,
-      path: `/repos/${owner}/${repo}/releases`,
+      path: `/repos/${owner}/${repo}/releases?per_page=100`,
       method: "GET",
     });
     data = data || [];
+    // filter for delete_pattern
+    const activeMatchedReleases = data.filter(({draft}) => !draft && tag_name.indexOf(deletePattern) !== -1)
```

Comment:
@dennistruemper  I see you merged my suggestion. But it is missing `tag_name` inside filter predicate parameters. Sorry!

Reply:
Also `activeMatchedReleases` not used in code. Please make that change also

Reply:
@dev-drprasad I fixed that now, sorry did not see that and didn't check. This time I ran it local, it works now for me.

We will get this working ;-)